src/IssueLabeler/IssueLabeler.csproj

```diff
@@ -4,6 +4,11 @@
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
     <UserSecretsId>401bcdfb-b94b-448f-80ac-36ea8a447a97</UserSecretsId>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Models\**" />
+    <EmbeddedResource Remove="Models\**" />
+    <None Remove="Models\**" />
+  </ItemGroup>
```

Comment:
```suggestion
```

Reply:
Not sure where this came from.

Reply:
VS is "helpful" this way when you delete stuff through the solution explorer.